client/src/pages/Configurations/ConfigActivate/index.tsx

```diff
@@ -1,55 +1,189 @@
+            heading: 'Configuration deactivated',
+            body: '',
+          });
+        },
+      }
+    );
+  }
 
   return (
     <div>
       <TitleContainer>
-        <Title>Condition name</Title>
-        {/* <Status version={response.data.active_version} /> */}
+        <Title>{configuration.data.display_name}</Title>
+        <Status version={configuration.data.active_version} />
       </TitleContainer>
       <NavigationContainer>
-        {/* <VersionMenu
-          id={response.data.id}
-          currentVersion={response.data.version}
-          status={response.data.status}
-          versions={response.data.all_versions}
+        <VersionMenu
+          id={configuration.data.id}
+          currentVersion={configuration.data.version}
+          status={configuration.data.status}
+          versions={configuration.data.all_versions}
           step="activate"
-        /> */}
+        />
         <StepsContainer>
           <Steps configurationId={id} />
         </StepsContainer>
       </NavigationContainer>
       <SectionContainer>
-        <ConfigurationTitleBar step="activate" />
-        <div>Configuration activation</div>
+        <ConfigurationTitleBar
+          step="activate"
+          condition={configuration.data.display_name}
+        />
+        <div className="max-w[80rem] mb-8 bg-white p-6">
+          <div className="mb-6">
+            <ul className="list-none">
+              <li>
+                <strong>Input: </strong>
+                {`eCRs from RCKMS with ${configuration.data.display_name} as a reportable condition`}
+              </li>
+              <li>
+                <strong>Output:</strong> Original + Refined eCR (in XML and HTML
+                formats)
+              </li>
+              <li>
+                <strong>Retained data:</strong>{' '}
+                {buildRetainedDataDisplay(configuration.data)}
+              </li>
+            </ul>
+          </div>
+          <hr className="text-gray-cool-20!" />
+          <ActivationButtons
+            configurationData={configuration.data}
+            isSuccess={isSuccess}
+            handleActivation={handleActivation}
+            handleDeactivation={handleDeactivation}
+            curVersion={configuration.data.version}
+            activeVersion={configuration.data.active_version}
+          ></ActivationButtons>
+        </div>
       </SectionContainer>
     </div>
   );
 }
+
+function buildRetainedDataDisplay(configurationData: GetConfigurationResponse) {
+  let codeSetDisplay = configurationData.code_sets
+    .map((c) => {
+      return `${c.display_name}`;
+    })
+    .join(', ');
+
+  // split out the last trailing comma
+  codeSetDisplay = `${codeSetDisplay.slice(0, codeSetDisplay.length)} code set(s)`;
+
+  const numCustomCodes = configurationData.custom_codes.length;
+  if (configurationData.custom_codes.length > 0) {
+    codeSetDisplay += `, ${numCustomCodes} custom code(s)`;
+  }
+
+  return codeSetDisplay;
+}
```

Comment:
👍  Looks good. My only suggestion here is just some super minor clean up. Double check me that this still works as it should!

```suggestion
function buildRetainedDataDisplay(configurationData: GetConfigurationResponse) {
  const codeSetNames = configurationData.code_sets
    .map((c) => c.display_name)
    .join(', ');
  const customCodeCount = configurationData.custom_codes.length;

  let result = `${codeSetNames} code set(s)`;

  if (customCodeCount > 0) {
    result += `, ${customCodeCount} custom code(s)`;
  }

  return result;
}
```